tests/test_phase_psf.py

```diff
@@ -261,6 +273,7 @@ def test_frozen_flow():
     with warnings.catch_warnings():
         warnings.simplefilter("ignore")
         aper = galsim.Aperture(diam=1, pupil_plane_size=20., pupil_plane_scale=20./dx)
+    screen.instantiate()
```

Comment:
Done.  I slightly changed the API of `wavefront()` to accept `None` as a valid argument for time, since that's how `_wavefront()` works.  I could have worked around this, but it seemed like a reasonable thing to allow.